master/internal/rm/kubernetesrm/informer.go

```diff
@@ -8,70 +8,36 @@ import (
 	"k8s.io/client-go/tools/cache"
 	watchtools "k8s.io/client-go/tools/watch"
 
-	"github.com/determined-ai/determined/master/pkg/actor/actors"
+	"github.com/sirupsen/logrus"
 
 	k8sV1 "k8s.io/api/core/v1"
 	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	typedV1 "k8s.io/client-go/kubernetes/typed/core/v1"
-
-	"github.com/determined-ai/determined/master/pkg/actor"
 )
 
-const defaultInformerBackoff = 5 * time.Second
```

Comment:
renaming this because it's not primarily used in `informers.go`

Reply:
update: removing this variable in this PR, as it wouldn't appear in  https://github.com/determined-ai/determined/pull/7256 anyway